app/src/main/java/com/example/musicroulette/utils/SpotifyUtils.java

```diff
@@ -0,0 +1,58 @@
+package com.example.musicroulette.utils;
+
+import android.net.Uri;
+
+import okhttp3.MultipartBody;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+
+public class SpotifyUtils {
```

Comment:
You're going to have to show me how you figured out how to build this haha

Reply:
Yeah lets meet up sometime this week (before friday) if everyone has time so I can explain what I did.